indico/modules/rb_new/client/js/components/modals/BlockingModal.jsx

```diff
@@ -229,14 +332,26 @@ class BlockingModal extends React.Component {
     };
 
     render() {
-        const {open, onClose} = this.props;
+        const {
+            open, onClose,
+            blocking: {blocked_rooms: blockedRooms, allowed, start_date: startDate, end_date: endDate, reason}
+        } = this.props;
+        const {mode} = this.state;
+        const props = mode === 'read' ? {onSubmit() {}} : {validate, onSubmit: this.processBlocking};
```

Comment:
`onSubmit` is apparently required and can't be undefined. Unfortunately wrapping it with the `FinalForm` is also necessary since when rendering the form I depend on some of the properties the rendering function receives.